digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -616,10 +630,18 @@ Returns empty result as Json message, not as page not found
     params.get("bbox").map { bbox =>
       val boundingRectangle = constructBoundingRectangle(bbox)
       validateBoundingBox(boundingRectangle)
-      if(user.isServiceRoadMaintainer())
-        mapLinearAssets(linearAssetService.withRoadAddress(linearAssetService.getByIntersectedBoundingBox(typeId, user.configuration.authorizedAreas, boundingRectangle, municipalities)))
-      else
-        mapLinearAssets(linearAssetService.withRoadAddress(linearAssetService.getByBoundingBox(typeId, boundingRectangle, municipalities)))
+      if(params("withRoadAddress") == "true"){
+        if(user.isServiceRoadMaintainer())

```

Comment:
Like you have this val 'withRoadAddress' you could replace -> if(params("withRoadAddress") == "true") by if(withRoadAddress == "true")

Reply:
If you do the modification to Boolean type like I say above, in this case you can use:

if(params("withRoadAddress").toBoolean) {
..........      
}